src/Symfony/Component/Console/Tests/Command/InvokableCommandTest.php

```diff
@@ -495,3 +495,5 @@ enum StringEnum: string
     case Image = 'image';
     case Video = 'video';
 }
+
+// @php-cs-fixer-ignore static_lambda Need to explicitly pass non-static lambdas to `setCode()`
```

Comment:
If we would make [this lambda](https://github.com/keradus/symfony/blob/69c01f522e4904f0d93296a452168cadb611250c/src/Symfony/Component/Console/Tests/Command/InvokableCommandTest.php#L40) static, test will start failing.

I appreciate any clue why, as it does not rely on $this.
Maybe sth we can even improve on Fixer itself...